MainProject/Models/Basics/Normal.cs

```diff
@@ -1,7 +1,11 @@
 ﻿namespace MainProject.Models.Basics
 {
-    public class Normal : Vector
+    public struct Normal
```

Comment:
Do we really need to make it structure? Now we won't be able to cast vector to normal and backwards